activerecord/test/cases/adapters/postgresql/test_arel_alias.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "support/connection_helper"
+require "models/posts"
+
+module ActiveRecord
+  class TestCreateUpdateDeleteAlias < ActiveRecord::AbstractMysqlTestCase
```

Comment:
Currently, the test classes overlap each other, so it is not testing all adapters correctly.

1. The test needs to extend `PostgreSQLTestCase`, not `AbstractMysqlTestCase`
2. The test class must end with `Test`, not start with it.
3. The test class should roughly match the file. e.g. `postgresql/arel_table_alias_test.rb`

```suggestion
  class PostgresqlArelTableAliasTest < ActiveRecord::PostgreSQLTestCase
```

Reply:
Really appreciate the feedback here!  Thanks for helping me get this formatted properly.  Just uploaded a version that implemented these properly.